django_snowflake/client.py

```diff
@@ -33,17 +34,17 @@ def settings_to_cmd_args_env(cls, settings_dict, parameters):
         if authenticator:
             args += ['--authenticator', authenticator]
         if client_session_keep_alive:
-            args += ['--client-session-keep-alive', client_session_keep_alive]
+            args += ['--client-session-keep-alive']
         if dbname:
             args += ['-d', dbname]
         if host:
             args += ['-h', host]
         if passcode:
             args += ['--mfa-passcode', passcode]
         if passcode_in_password:
-            args += ['--mfa-passcode-in-password', passcode_in_password]
```

Comment:
Was this removal (`passcode_in_password`) accidental?

Reply:
No, see the second commit of the PR (it's not related to the private key stuff). This and client-session-keep-alive are boolean options that don't take arguments.